blocksuite/affine/block-database/src/data-source.ts

```diff
@@ -180,13 +180,25 @@ export class DatabaseBlockDataSource extends DataSourceBase {
 
   propertyAdd(insertToPosition: InsertToPosition, type?: string): string {
     this.doc.captureSync();
+
+    const metaConfig = this.propertyMetaGet(
+      type ?? propertyPresets.multiSelectPropertyConfig.type
+    );
```

Comment:
`propertyMetaGet` method looks like this
```typescript
  propertyMetaGet(type: string): PropertyMetaConfig {
    return databaseBlockAllPropertyMap[type];
  }
```

It returns the property configuration object corresponding the type.